### ISSUE TITLE
Adding the documentation for serverpod insights

### ISSUE THREAD

**Issue by Cavin6080** [serverpod/serverpod#600]
As **serverpod 1** has been released, I downloaded the servepod insights tool. Where I have to add a project to it.

But as a user, I didn't find any guides that which file or folder I have to select in order to add to the project in the serverpod. I also dig deep into the documentation but I didn't find any documentation of it.

It would be great if the **documentation of serverpod insights** would be added in the docs or in the software itself.

**Comment by Cavin6080** [comment]
I managed to figure out how to select projects and perform logs. I would like to write official documentation about it.

**Comment by yahu1031** [comment]
@Cavin6080 Please 🙏🏻 If you can do it, write one in https://github.com/serverpod/serverpod_docs and raise a PR. That would be helpful.

**Comment by vlidholt** [comment]
Yeah, Insights are still in beta, so we may be adding both better documentation and perhaps some guidance when you first run the app.
